uktrade/data-hub-frontend
# Reactify OMIS invoice details form

## Description of change

The form for editing the invoice details for an OMIS order has been reactified.

## Test instructions

Go to an OMIS order and edit the invoice details. Everything should work as before.

## Screenshots

### Before

![Screenshot 2023-10-16 at 16-51-03 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/e6611cf2-c577-4b8e-b353-8164e5435b6b)

### After

![Screenshot 2023-10-16 at 16-50-26 Edit invoice details - JTJ148_23 - Orders (OMIS) - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/59e50820-fe5c-4e10-934e-6bb42545592d)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
